RX_FSK/data/livemap.js

```diff
@@ -321,7 +321,7 @@ headtxt = function(data,stat) {
     var datetime = m.getUTCFullYear() + "-" + az(m.getUTCMonth()+1) + "-" + az(m.getUTCDate()) + "T" +
       az(m.getUTCHours()) + ":" + az(m.getUTCMinutes()) + ":" + az(m.getUTCSeconds()) + "Z";
     var url = 'https://predict.cusf.co.uk/api/v1/';
-    url += '?launch_latitude='+data.lat + '&launch_longitude='+data.lon;
+    url += '?launch_latitude='+data.lat + '&launch_longitude='+fix_lon(data.lon);
```

Comment:
I can have a closer look later but I guess here you need the fix in the opposite direction (convert local -180..180 to 0..360 as input parameter for the tawhiri API?

Reply:
fix_lon works in both directions, but please have a closer look as i never had such problems here in germany.